themes/_core.css

```diff
@@ -437,10 +437,11 @@ body.close .content {
     transform: translateX(16rem);
   }
   body.close .sidebar-toggle {
-    background-color: rgba(255,255,255,0.8);
+    /* background-color: rgba(255, 255, 255, 0.8);
     transition: 1s background-color;
     width: 0rem;
-    padding: 10px;
+    padding: 10px; */
```

Comment:
It's always better to delete than comment-out: it avoids clutter and deletions are always recoverable.